packages/core/src/internal/utils/events.ts

```diff
@@ -9,6 +9,10 @@ export function stopEvent(event: Event) {
   event.stopPropagation();
 }
 
+function isNode() {
```

Comment:
Good job making this a utility. But I don't know if `events.ts` is a good home for it. We have a similar check for SSR called `isBrowser` that lives in `exists.ts`. 

It may make sense to take `isBrowser` and `isNode` and pull them into their own utility file. Maybe something like `environment.ts` or `platform.ts`? Whichever name makes sense.

These are internal files/utilities so moving them around like that is not considered a breaking change. Only power users are going to be aware of them and use them extensively.

Reply:
It actually already was a utility, I just moved it from one file to another 😆

Is it overkill to move it to a utility file is this is the only place it's used? I can see the benefits of having it live next to `isBrowser` though.

Reply:
I don't think it's overkill. I was surprised that `isBrowser` is in `exists.ts`. That's weird. It's a lot easier to track these utilities down if they live somewhere sensible. The only caveat with that approach is avoiding a circular module import.